operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -44,7 +44,10 @@ serviceAccount:
 # -- Configuration is loaded from the secret called 'externalConfigSecretName'. If 'useExternalConfig' is true, then the configuration is not generated, just consumed.
 useExternalConfig: false
 
-# -- Name of the secret that contains the configuration (used for naming even if config is internal).
+# -- What kind of object stores the configuration, choice of ConfigMap or Secret. Note that ConfigMap alone is not secure and you should inject passwords, keys via the global.extraEnvFrom mechanism.
```

Comment:
Can you elaborate the docs on what the "global.extraEnvFrom mechanism" is? I don't see it explained anywhere in the docs.

Reply:
I don't see a reason not to include the explanation in the docstrings as well and it makes the generated helm-docs documentation sufficient to deal with the chart. I don't want this to be a blocker, because I haven't spent too much time thinking about how the docs should ultimately look.

Reply:
expanded on it, but I think I'll stop here and we'll make it nice with the doc team

Reply:
I think what you have is sufficient 👍